pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/spec/SegmentGenerationJobSpec.java

```diff
@@ -139,6 +139,11 @@ public class SegmentGenerationJobSpec implements Serializable {
    */
   private String _authToken;
 
+  /**
+   * Create a separated metadata only tar gz file to reduce the data transfer of segment metadata push job.
+   */
+  private boolean _createMetadataTarGz = true;
```

Comment:
I feel that we should disable it by default as for now to avoid potential problems. We can enable it by default after we have fully tested it in some envs.

Reply:
will update the default